_docs/8_BlockExplorerSquid/docker-compose.yml

```
@@ -0,0 +1,62 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:14
+    volumes:
+      # replace VOLUME_NAME with your volume name
+      - VOLUME_NAME/postgresql/data:/var/lib/postgresql/data
```

I'm fairly certain you meant this:
```suggestion
      - /VOLUME_NAME/postgresql/data:/var/lib/postgresql/data
```

Since there is no volume defined in this file.